Add link to the Business Central Technical Validation Checklist (#73600)
Azure/azure-content: articles/marketplace/partner-center-portal/create-new-business-central-offer.md

This page defines the technical details used to connect to your offer. This connection enables us to provision your offer for the end customer if they choose to acquire it.

> It is required that the extensions submitted for your offer meet the requirements specifed in the [Technical Validation Checklist](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-checklist-submission).

Reviewer: ```suggestion
Extensions submitted for your offer must meet the requirements specified in the [Technical Validation Checklist](/dynamics365/business-central/dev-itpro/developer/devenv-checklist-submission).
```
